frontend/simple/js/state.js

```diff
@@ -99,19 +117,45 @@ const getters = {
   unreadMessageCount (state) {
     let mailboxContract = store.getters.mailboxContract
     return mailboxContract && mailboxContract.messages.filter((msg) => !msg.read).length
+  },
+  // Logged In user's identity contract
+  identity (state) {
```

Comment:
Edit: nope, I was wrong. See how misleading these names can be? This seems to return their identity **contract ID**, which is not what the comment says either.

Let's rename this to `currentUserContract`, since that seems most convenient, and have it return that, and then get rid of the attribute-specific methods below, which go into too much detail for this file, which doesn't need a function-per-attribute (and in this case, two functions per attribute). Way overkill.